src/app/catalog/components/badge-catalog/badge-catalog.component.html

```diff
@@ -119,7 +119,9 @@ <h3 class="u-text-h3-semibold u-margin-bottom2x u-margin-top6x u-text-dark1">
 					</div>
 				</ng-container>
 			</ng-container>
-			<ng-container *ngIf="groupBy == 'Kategorie'">
+			<!-- ToDo: test it with real data -->
```

Comment:
Is this TODO still open?

Reply:
It's just to remember to test it with real data. But should be done for this issue.